packages/react-cards/src/components/Card/CardSection/CardSection.view.tsx

```diff
@@ -1,12 +1,11 @@
 /** @jsx withSlots */
-import * as React from 'react';
 import { withSlots, getSlots } from '@uifabric/foundation';
 import { Stack } from '@fluentui/react/lib/Stack';
 import { ICardSectionComponent, ICardSectionProps, ICardSectionSlots } from './CardSection.types';
 
 export const CardSectionView: ICardSectionComponent['view'] = props => {
   const { children, ...rest } = props;
-  if (React.Children.count(children) < 1) {
+  if (children === undefined) {
```

Comment:
```suggestion
  if (children == null) {
```

Reply:
we should do the check like this so it covers both `undefined` and `null`

Reply:
Sure, tho I wouldn't except `null` ever to be passed. React passes `undefined`. `null` would only be passed by library consumers.